batch_job/src/batch_job_work_queue.c

```diff
@@ -108,6 +108,10 @@ static batch_job_id_t batch_job_wq_wait (struct batch_queue * q, struct batch_jo
 		{
 			return -1;
 		}
+
+		char *transactions = string_format("%s.transactions", q->logfile);
+		work_queue_specify_transactions_log(q->data, transactions);
+		free(transactions);
```

Comment:
More of a meta question, but is there a reason we wait to specify the log and transaction log until batch_job_*_wait, as opposed to batch_job_*_create?